### PR TITLE
OFFIS extensions and analysis

### PR DESCRIPTION
**Pull request by jansbecker**
Added content with updated timing requirements

* AMALTHEA models in sub-folders
* Extended TIM
* Updated requirements
* READMEs

Contributions by OFFIS

### CODE REVIEW

File: org.panorama-research.mobstr.tim/model/mobstr-tim.xcore [hunk 1]
```diff
@@ -244,4 +245,26 @@ class RefinesItemDefinition extends DirectedTraceLink {
  class RefersToAnalysisResult extends DirectedTraceLink {
  	refers Element [1] source // Element -- Safety Case
  	refers Element [1..*] target // Element -- Failure in FTA
- }
\ No newline at end of file
+ }
+ 
+ /*******************************
+  *       OFFIS Extensions
+  *******************************/
```
Comment (steghoja) [inline org.panorama-research.mobstr.tim/model/mobstr-tim.xcore:252]: Is this comment meaningful to outsiders down the line?

File: org.panorama-research.mobstr.tim/src/org/panorama/research/mobstr/tim/MobstrTIMAdapter.java [hunk 2]
```diff
@@ -78,6 +78,10 @@ public EObject createModel() {
 			traceTypes.add(TimPackage.eINSTANCE.getRelatedTo());
 			traceTypes.add(TimPackage.eINSTANCE.getSpecifiedByRequirement());
 			traceTypes.add(TimPackage.eINSTANCE.getUsesAsEvidence());
+			
+			// OFFIS Extensions
```
Comment (steghoja) [inline org.panorama-research.mobstr.tim/src/org/panorama/research/mobstr/tim/MobstrTIMAdapter.java:82]: Same as above. Is this a meaningful comment if someone who is not familiar with the history of the project looks at that code in a year or two?

**Review by steghoja — COMMENTED** [review]
Thanks for your contribution, Jan! Please provide a more meaningful commit message that describes what the extension and analysis actually consists of. This will allow others to gauge whether the new content is interesting for them. I am also not sure how helpful the code comments "OFFIS extension" will be for future users of the dataset and encourage you to remove them.

File: org.panorama-research.mobstr.tim/model/mobstr-tim.xcore (hunk 1 above)
Comment (jansbecker) [reply org.panorama-research.mobstr.tim/model/mobstr-tim.xcore:252]: I removed that comment

File: org.panorama-research.mobstr.tim/src/org/panorama/research/mobstr/tim/MobstrTIMAdapter.java (hunk 2 above)
Comment (jansbecker) [reply org.panorama-research.mobstr.tim/src/org/panorama/research/mobstr/tim/MobstrTIMAdapter.java:82]: I have removed that comment

**Pull request merged**